plugins/inputs/chrony/README.md

```diff
@@ -53,6 +53,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   # socket_perms = "0660"
 ```
 
+## Probing
+
+This plugin supports probing through the following configuration:
+
+```toml
+startup_error_behavior = "probe"
+```
+
```

Comment:
Please put this _before_ the `Configuraton` section but below `Global configuration options`! It would also be nice to make this an include-able text so we can include it in all plugins that supports probing. Alternatively, You can keep the text here but describe what is done for probing...

Reply:
Added includable docs and change the location like you suggested 👍🏻